src/Transport/Addressing/Sanitization/Strategies/ThrowOnFailedValidation.cs

```diff
@@ -103,6 +107,7 @@ internal ThrowOnFailedValidation(ReadOnlySettings settings)
             }
         }
 
+        /// <summary></summary>
```

Comment:
empty

Reply:
In this case fully agree, it should be documented. `Sanitize()` here has a hidden behavior that is not obvious. Done.